c_examples/mem-dump.c

```diff
@@ -47,4 +47,4 @@ int main(int argc, char* argv[]) {
     dump_memory(driver, argv[1]);
     microvmi_destroy(driver);
     return 0;
-}
+}
```

Comment:
Please install the editorconfig plugin so we don't see such unnecessary whitespace changes again.